salt/config/schemas/common.py

```diff
@@ -9,7 +9,7 @@
     Common salt configuration schemas
 '''
 
-# Import Pythosn libs
+# Import Pythons libs
```

Comment:
Thanks @sadminriley and welcome to salt! Since we're fixing this line anyway, I think this should just be `Python` instead?